src/optional.h

```diff
@@ -0,0 +1,1100 @@
+// Copyright (C) 2011 - 2012 Andrzej Krzemienski.
+//
+// Use, modification, and distribution is subject to the Boost Software
+// License, Version 1.0. (See below or copy at http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
I've changed the file header to include the license directly (see https://github.com/akrzemi1/Optional/blob/master/optional.hpp for comparison). I'm not well-versed in licensing, so please double-check if this reasonably waterproof.